### PR TITLE
4199 sweep along vector issue

### PR DESCRIPTION
**Pull request by amichel0205**
This PR is to fix issue [#4199.](https://github.com/ansys/pyaedt/issues/4199)

Fix issue in sweep_along_vector() when managing list of object as input.

Update UT test_47_sweep_along_vector() additional lines

### CODE REVIEW

File: pyaedt/modeler/cad/Primitives.py [hunk 1]
```diff
@@ -2863,8 +2863,8 @@ def sweep_along_vector(self, objid, sweep_vector, draft_angle=0, draft_type="Rou
 
         Returns
         -------
-        bool
-            ``True`` when successful, ``False`` when failed.
+        pyaedt.modeler.cad.object3d.Object3d, list
```
Comment (Samuelopez-ansys) [inline pyaedt/modeler/cad/Primitives.py:2866]: It should be:

```
:class:`pyaedt.modeler.cad.object3d.Object3d` or list of :class:`pyaedt.modeler.cad.object3d.Object3d`
            Objects created.
```

Comment (amichel0205) [reply pyaedt/modeler/cad/Primitives.py:2866]: I'll correct it

Comment (amichel0205) [reply pyaedt/modeler/cad/Primitives.py:2866]: Like this?
image

File: pyaedt/modeler/cad/Primitives.py [hunk 2]
```diff
@@ -2863,8 +2863,8 @@ def sweep_along_vector(self, objid, sweep_vector, draft_angle=0, draft_type="Rou
 
         Returns
         -------
-        pyaedt.modeler.cad.object3d.Object3d, list
-            3D object, list of 3D object.
+        :class:`pyaedt.modeler.cad.object3d.Object3d` or list of :class:`pyaedt.modeler.cad.object3d.Object3d`
+            Objects created.
```
Comment (PipKat) [inline pyaedt/modeler/cad/Primitives.py:2867]:
```suggestion
            One or more objects created.
```

**Review by Samuelopez-ansys — APPROVED** [review]
LGTM

**Pull request merged**